sbol_utilities/sbol3_sbol2_conversion.py

```diff
@@ -1,3 +1,4 @@
+import sbol2.model
```

Comment:
This import should not be needed, given the `from sbol2` line below.
In fact, we should probably remove that one too since it makes the references in the code more ambiguous.